examples/aws/ingress/values.yaml

```diff
@@ -0,0 +1,32 @@
+# This example shows how to deploy and expose GraphDB with AWS Load Balancer Controller Ingress.
+
+configuration:
+  externalUrl: graphdb-example-dns-name.com # Change this to your ALB DNS name or Route53 if you use it.
+
+service:
+  enabled: true
+  type: NodePort
```

Comment:
You don't need this for the Ingress scenario

Reply:
Forgot to remove it.